lib/slurp/application.ex

```diff
@@ -20,5 +20,17 @@ defmodule Slurp.Application do
     Enum.each(new_head_subscriptions, &Slurp.NewHeads.Subscriptions.put/1)
     {:ok, log_subscriptions} = Slurp.Logs.Subscriptions.from_config(blockchains)
     Enum.each(log_subscriptions, &Slurp.Logs.Subscriptions.put/1)
+
+    with b <- blockchains
```

Comment:
Can you add a new `start_phase` for this as it's not hydration. Something like `:blockchains_and_subscriptions`?

How do you feel about naming the `b` variable something like `ids`?

Reply:
Sure, Ill add a new start phase.